pkg/ottl/ottlfuncs/README.md

```diff
@@ -1033,6 +1034,32 @@ Examples:
 
 **Note:** According to the National Institute of Standards and Technology (NIST), MD5 is no longer a recommended hash function. It should be avoided except when required for compatibility. New uses should prefer a SHA-2 family function (e.g. SHA-256, SHA-512) whenever possible.
 
+### MarshalKeyValue
```

Comment:
I believe this is the first function we're introducing that does the opposite of the `Parse*` functions.

I'd like to discuss if we should use the term `Marshal` for these type of functions or use something like `To*String`. I ask because we didn't use `Unmarshal*` for the parsers.

Reply:
`ToKeyValueString` makes sense to me.

Reply:
This makes sense. Renaming